model_analyzer/config/generate/generator_factory.py

```diff
@@ -22,20 +22,19 @@ class ConfigGeneratorFactory:
     """
 
     @staticmethod
-    def create_model_config_generator(config,
-                                      model,
-                                      client,
-                                      variant_name_manager,
-                                      default_only=False):
+    def create_model_config_generator(config, model, client,
+                                      variant_name_manager, default_only,
+                                      early_exit_enable):
```

Comment:
Should this have a comment describing the parameters? Or is that not needed because this is the factory?